packages/types/src/lib/interfaces.ts

```diff
@@ -272,7 +272,6 @@ export interface BaseJsonPkpSignRequest {
 export interface JsonPkpSignSdkParams extends BaseJsonPkpSignRequest {
   pubKey: string;
   sessionSigs: SessionSigsMap;
-  authMethods?: AuthMethod[];
```

Comment:
Removing authMethods form executeJs?

Reply:
no, it's `pkpSign`, but should we still accept `authMethod`?

Reply:
No pkpSign shouldn't accept authMethods either

Reply:
Cool! so no change required for this interface